### PR TITLE
Refresh session after successful purchase

### PR DESCRIPTION
**Pull request by aninternetof**
This PR changes two things:
* After a successful purchase, the user is returned to the purchase page.
  * I changed the text that shows to be more success-y.
* Refreshes the jwt upon arriving on the purchase page

### CODE REVIEW

**Comment by cabarnes** [comment]
This will need to be updated to work with: https://github.com/rovercode/rovercode-web/pull/331

**Comment by aninternetof** [comment]
[This commit](https://github.com/rovercode/rovercode-ui/pull/290/commits/6021d919c9f3a394acb7928bd8b8632547f09f5a) seems to make the refresh work with the new refresh API.

**Comment by aninternetof** [comment]
I noticed one thing about the refresh action (both how it was and how it is in this PR). Should we be saving the token(s) from the refresh response to the cookie?

File: webpack.local.js [hunk 1]
```diff
@@ -10,8 +10,8 @@ module.exports = merge(common, {
     historyApiFallback: true,
     contentBase: './',
     proxy: {
-      '/api': 'http://localhost:8000',
-      '/jwt': 'http://localhost:8000',
+      '/api': 'http://localhost:8001',
```
Comment (cabarnes) [inline webpack.local.js:13]: There needs to be a corresponding PR for the API with this change

**Comment by cabarnes** [comment]
@aninternetof Yes, we should be saving the new credentials to the cookie. That was missed before

File: src/components/Purchase.js [hunk 2]
```diff
@@ -152,7 +153,9 @@ class Purchase extends Component {
                 <FormattedMessage
                   id="app.friends_family.already_purchased"
                   description="Messages explaining that the user already purchased the Individual Plan"
-                  defaultMessage="You have already purchased the Individual Plan for this account"
+                  defaultMessage="You have successfully purchased the Individual Plan for this account!
+                                  Your Rover should arrive in 5-10 business days. While you wait, feel
```
Comment (cabarnes) [inline src/components/Purchase.js:157]: Probably missing `free` after `feel`?

**Comment by aninternetof** [comment]
Any ideas on how I can set the cookie within the action or reducer? I'd rather not have to save the tokens to the store and set the cookie in component logic.

**Comment by aninternetof** [comment]
Actually, redux actions probably aren't supposed to have side effects like setting a cookie, huh? I can add the tokens to the store and put them in the cookie in a component.

File: webpack.local.js (hunk 1 above)
Comment (aninternetof) [reply webpack.local.js:13]: Yes, I've got that ready.

**Review by cabarnes — COMMENTED** [review]
There's still the typo in the message that I had commented previously

File: src/components/Purchase.js (hunk 2 above)
Comment (aninternetof) [reply src/components/Purchase.js:157]: Fixed.

**Review by cabarnes — APPROVED** [review]
Looks good!

**Pull request merged**